app/controllers/home_controller.rb

```diff
@@ -1,5 +1,7 @@
 class HomeController < ApplicationController
   def index
-    
+    if current_user 
```

Comment:
instead of making a new `@user` variable in this controller I would recommend just using the `current_user` in the index.html.erb.